test_fms/diag_integral/test_diag_integral.F90

```diff
@@ -96,6 +96,24 @@ program test_diag_integral
   call test_sum_diag_integral_field !< compare read in values to the expected values.
 
 contains
+  !-------------------------------------
```

Comment:
Should this test be updated like the other one to pass in the variables to each routine?

The only real change I'm seeing in this test is moving the `intialize_arrays` subroutine up in the file, which shouldn't change anything technically, but maybe ifx thinks otherwise.